### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by jjm2473**

### CODE REVIEW

File: grails-app/services/streama/TheMovieDbService.groovy [hunk 1]
```diff
@@ -159,8 +161,7 @@ class TheMovieDbService {
       apiCacheData["$type:$name"] = data
     }
     catch(e) {
-      HttpURLConnection conn = url.openConnection()
-      throw new Exception("TMDB request failed with statusCode: " + conn?.responseCode + ", responseMessage: " + conn?.responseMessage + ", url: " + requestUrl)
```
Comment (dularion) [inline grails-app/services/streama/TheMovieDbService.groovy:163]: why remove this exception text? its pretty useful for the content manager to know whats going on

File: build.gradle [hunk 2]
```diff
@@ -80,14 +80,15 @@ dependencies {
...
-    compile "org.grails.plugins:mail:2.0.0"
+    compile "org.grails.plugins:mail:3.0.0"
     compile "org.grails.plugins:grails-spring-websocket:2.4.1"
     compile "eu.bitwalker:UserAgentUtils:1.21"
 
     // runtime
     runtime "com.h2database:h2"
     runtime 'mysql:mysql-connector-java:5.1.36'
     runtime "com.bertramlabs.plugins:asset-pipeline-grails:2.11.6"
+    runtime 'javax.xml.bind:jaxb-api:2.3.0'
```
Comment (dularion) [inline build.gradle:91]: what do you need this library for?

File: build.gradle [hunk 3]
```diff
@@ -80,14 +80,15 @@ dependencies {
     compile "org.hibernate:hibernate-core:5.1.2.Final"
     compile "org.hibernate:hibernate-ehcache:5.1.2.Final"
     compile 'org.grails.plugins:spring-security-core:3.0.4'
-    compile "org.grails.plugins:mail:2.0.0"
+    compile "org.grails.plugins:mail:3.0.0"
```
Comment (dularion) [inline build.gradle:83]: 👍

File: grails-app/services/streama/TheMovieDbService.groovy (hunk 1 above)
Comment (jjm2473) [reply grails-app/services/streama/TheMovieDbService.groovy:163]: I think:
1 . the `e` contains statuscode;
2. calling url.openConnection() twice not a good solution.

File: build.gradle (hunk 2 above)
Comment (jjm2473) [reply build.gradle:91]: run streama with java 11 will failed for `javax.xml.bind` not found .

for details:
java 9 remove javax.xml.bind from jre, still can be added as module;
java 11 totally remove javax.xml.bind from jre.

File: grails-app/services/streama/TheMovieDbService.groovy (hunk 1 above)
Comment (dularion) [reply grails-app/services/streama/TheMovieDbService.groovy:163]: all right lets try :)

**Pull request merged**